### ISSUE TITLE
No all the classes can be decompiled.

### ISSUE THREAD

**Issue by liangdefeng** [cnfree/Eclipse-Class-Decompiler#22]
1. OS
    CentOS 7.3
2. JDK Edition and version.
    openjdk version "1.8.0_131"
    OpenJDK Runtime Environment (build 1.8.0_131-b12)
    OpenJDK 64-Bit Server VM (build 25.131-b12, mixed mode)
3. Eclipse:
    Eclipse Java EE IDE for Web Developers.
    Version: Neon.3 Release (4.6.3)
    Build id: 20170314-1500
4. Eclipse Class Decompiler 
    Version: 2.9.4
    Build id: 20170703
5. Eclipse Class Decompiler setting.
![image](https://user-images.githubusercontent.com/20069277/27813198-d2c46d7e-60a6-11e7-9b05-79e050c2a5c8.png)

6. Observations.
    Project description: The MyPlugin Eclipse project is a project created using the Eclipse wizard. I added an Oracle JDBC driver's jar to the project for testing purpose.

Observations.
1) The classes in jars in the red rectangle can be decompiled.
2) The classes in jars in the green rectangle can't be decompiled.
![image](https://user-images.githubusercontent.com/20069277/27813237-1f54e146-60a7-11e7-892d-81a988ea426c.png)
3) This issue appears in the recent releases, but I am not sure exactly when it was introduced. I have another environment where I have eclipse Neon.3 Release (4.6.3) and Eclipse Class Decompiler 2.9.0. It works well.

@cnfree, could you help to see where is the problem.

**Comment by liangdefeng** [comment]
I find where the problem and resolve the problem now. I close the issue.

I have two users in the environment. I have used the plug-in under one account, and I switched to another account.

When I used the first account, it will create a folder /tmp/.org.sf.feeling.decompiler. The owner of the folder belongs to the account.

After I switched to another user, as the account doesn't has right to write the folder /tmp/.org.sf.feeling.decompiler. Thus some classes can't be compiled,for their  decompiled result is supposed to be found in the the folder /tmp/.org.sf.feeling.decompiler.

After I deleted the folder /tmp/.org.sf.feeling.decompiler, it works again!
